src/Validation/ConstraintViolation.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+
+namespace WMDE\Fundraising\Frontend\Validation;
+
+/**
+ * @license GNU GPL v2+
+ * @author Gabriel Birke < gabriel.birke@wikimedia.de >
+ */
+class ConstraintViolation {
+	private $value;
+	private $message;
+	private $source;
+	private $constraint;
+
+	/**
+	 * @param mixed $value The value that caused this violation
+	 * @param string $message Error message
+	 * @param mixed $constraint The validator instance that created this
+	 * @param string $source Class name or Class.Field name
+	 */
+	public function __construct( $value, string $message, $constraint = null, string $source = '' ) {
```

Comment:
What's the deal with this highlighting of the empty string?

![confused](https://cloud.githubusercontent.com/assets/146040/12309217/bdac2af8-ba49-11e5-9db0-e2ceb8b8c332.jpg)